packages/components/src/Dropzone/sub-components/OnDragMessage/OnDragMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Paragraph } from "@kaizen/typography"
+
+export const OnDragMessage = (): JSX.Element => (
+  <div className="flex flex-col items-center justify-center">
```

Comment:
You shouldn't need to use `flex` for this. Even if you remove all the spans, a `text-center` should align nicely.